integration/__tests__/app.tests.ts

```diff
@@ -8,8 +8,9 @@ import { Application } from "spectron";
 import * as util from "../helpers/utils";
 import { spawnSync } from "child_process";
 
-const describeif = (condition: boolean) => condition ? describe : describe.skip;
+// eslint-disable-next-line unused-imports/no-unused-vars-ts
 const itif = (condition: boolean) => condition ? it : it.skip;
```

Comment:
perhaps just remove `const itif = (condition: boolean) => condition ? it : it.skip;`

Reply:
Better Idea, have both be exported in some "utils/for-tests.ts" file.